### DISCUSSION TITLE
Main.css not outputting

### DISCUSSION THREAD

**Discussion by ghost** [community/community#34107] [Other Feature Feedback, Questions, & Ideas]
Please I have issues in tailwindcss, after configuration of postcss, anything written on main.css don't reflect, please where do think I got it wrong.
Thanks and my first time of being here

**Comment by airtower-luna** [comment]
It looks like you're looking for help with a software hosted on GitHub, maybe https://github.com/tailwindlabs/tailwindcss? In that case you should use the discussions _in that repository_ so the developers and other people familiar with it see your question. You might want to add more details about what you're doing so it's easier to help you, too. :slightly_smiling_face:

**Comment by ihnatenkoo** [comment]
Hello, can you send link to the repository? =)
